ui/resolution_selection.js

```diff
@@ -79,40 +78,65 @@ shaka.ui.ResolutionSelection = class extends shaka.ui.SettingsMenu {
 
 
     this.eventManager.listen(this.player, 'loading', () => {
-      this.updateResolutionSelection_();
-      this.updateResolutionLabels_();
+      this.updateSelection_();
+      this.updateLabels_();
     });
 
     this.eventManager.listen(this.player, 'unloading', () => {
-      this.updateResolutionSelection_();
-      this.updateResolutionLabels_();
+      this.updateSelection_();
+      this.updateLabels_();
     });
 
     this.eventManager.listen(this.player, 'variantchanged', () => {
-      this.updateResolutionSelection_();
-      this.updateResolutionLabels_();
+      this.updateSelection_();
+      this.updateLabels_();
     });
 
     this.eventManager.listen(this.player, 'trackschanged', () => {
-      this.updateResolutionSelection_();
-      this.updateResolutionLabels_();
+      this.updateSelection_();
+      this.updateLabels_();
     });
 
     this.eventManager.listen(this.player, 'abrstatuschanged', () => {
-      this.updateResolutionSelection_();
-      this.updateResolutionLabels_();
+      this.updateSelection_();
+      this.updateLabels_();
     });
 
     this.eventManager.listen(this.player, 'adaptation', () => {
-      this.updateResolutionLabels_();
+      this.updateSelection_();
+      this.updateLabels_();
     });
 
-    this.updateResolutionSelection_();
+    this.updateSelection_();
   }
 
   /** @private */
-  updateResolutionLabels_() {
-    const tracks = this.player.getVariantTracks() || [];
+  updateLabels_() {
+    const abrEnabled = this.player.getConfiguration().abr.enabled;
+    if (this.player.isAudioOnly()) {
+      if (this.overflowQualityMark) {
+        this.overflowQualityMark.textContent = '';
+        this.overflowQualityMark.style.display = 'none';
+      }
+      const audioTracks = this.player.getVariantTracks() || [];
```

Comment:
Why don't use `getAudioTracks()`?

Reply:
To keep the same old code, since I haven't made any changes here. Also, AudioTrack doesn't have bandwidth, which is what we're using here.